SPECS/emacs/emacs.spec

```diff
@@ -62,6 +77,38 @@ rm %{buildroot}%{_datadir}/applications/*.desktop
 
 mkdir -p %{buildroot}%{_datadir}/emacs/site-lisp/site-start.d
 
+# Install rpm macro definition file
+mkdir -p %{buildroot}%{_rpmconfigdir}/macros.d
+install -p -m 0644 macros.emacs %{buildroot}%{_rpmconfigdir}/macros.d/
+
+mkdir -p %{buildroot}%{site_lisp}
+install -p -m 0644 %SOURCE1 %{buildroot}%{site_lisp}/site-start.el
+install -p -m 0644 %SOURCE2 %{buildroot}%{site_lisp}
+
+# This solves bz#474958, "update-directory-autoloads" now finally
+# works the path is different each version, so we'll generate it here
+echo "(setq source-directory \"%{_datadir}/emacs/%{version}/\")" \
+ >> %{buildroot}%{site_lisp}/site-start.el
+
+mkdir -p %{buildroot}%{site_lisp}/site-start.d
+
+#
+# Create file lists
+#
+rm -f *-filelist {common,el}-*-files
+ 
+( TOPDIR=${PWD}
+  cd %{buildroot}
+ 
+  find .%{_datadir}/emacs/%{version}/lisp .%{site_lisp} \
+    \( -type f -name '*.elc' -fprint $TOPDIR/common-lisp-none-elc-files \) -o \( -type d -fprintf $TOPDIR/common-lisp-dir-files "%%%%dir %%p\n" \) -o \( -name '*.el.gz' -fprint $TOPDIR/el-bytecomped-files -o -fprint $TOPDIR/common-not-comped-files \)
+ 
+)
+
+# Put the lists together after filtering  ./usr to /usr
+sed -i -e "s|\.%{_prefix}|%{_prefix}|" *-files
+grep -vhE '%{site_lisp}(|/(default\.el|site-start\.d|site-start\.el))$' {common,el}-*-files > common-filelist
+
```

Comment:
I think the file list is never used. Might as well just drop this whole logic. Or go the other way and use it to simplify the `%files` section.

Reply:
Removed it